internal/crypto/ptls/secure.go

```diff
@@ -21,6 +21,7 @@ const secureServingOptionsMinTLSVersion = "VersionTLS13"
 // by tls.Config.
 const SecureTLSConfigMinTLSVersion = tls.VersionTLS13
 
+// Secure or Default should be the only way to create a tls.Config within any component of Pinniped.
```

Comment:
I didn't understand this comment in this context.

Reply:
I've updated the comments